src/shared/gpt.c

```diff
@@ -15,20 +15,29 @@
 #endif
 
 #define _GPT_ARCH_SEXTET(arch, name)                                   \
-        { GPT_ROOT_##arch,              "root-" name               },  \
-        { GPT_ROOT_##arch##_VERITY,     "root-" name "-verity"     },  \
-        { GPT_ROOT_##arch##_VERITY_SIG, "root-" name "-verity-sig" },  \
-        { GPT_USR_##arch,               "usr-" name                },  \
-        { GPT_USR_##arch##_VERITY,      "usr-" name "-verity"      },  \
-        { GPT_USR_##arch##_VERITY_SIG,  "usr-" name "-verity-sig"  }
+        { GPT_ROOT_##arch,              "root-" name,               ARCHITECTURE_##arch },  \
+        { GPT_ROOT_##arch##_VERITY,     "root-" name "-verity",     ARCHITECTURE_##arch },  \
+        { GPT_ROOT_##arch##_VERITY_SIG, "root-" name "-verity-sig", ARCHITECTURE_##arch },  \
+        { GPT_USR_##arch,               "usr-" name,                ARCHITECTURE_##arch },  \
+        { GPT_USR_##arch##_VERITY,      "usr-" name "-verity",      ARCHITECTURE_##arch },  \
+        { GPT_USR_##arch##_VERITY_SIG,  "usr-" name "-verity-sig",  ARCHITECTURE_##arch }
 
 const GptPartitionType gpt_partition_type_table[] = {
         _GPT_ARCH_SEXTET(ALPHA,       "alpha"),
         _GPT_ARCH_SEXTET(ARC,         "arc"),
         _GPT_ARCH_SEXTET(ARM,         "arm"),
         _GPT_ARCH_SEXTET(ARM64,       "arm64"),
         _GPT_ARCH_SEXTET(IA64,        "ia64"),
-        _GPT_ARCH_SEXTET(LOONGARCH64, "loongarch64"),
+
+        /* TODO: Replace with `_GPT_ARCH_SEXTET(LOONGARCH64, "loongarch64")` once
```

Comment:
please `//` stye comments for this.

In the coding style somewhere it suggests to use `/* */` style comments for explanatory stuff, and `//` for FIXME-style comments... This is the latter.

Makes it easier to find/highlight